ocs_ci/deployment/metallb.py

```diff
@@ -103,7 +110,7 @@ def create_catalog_source(self):
         # replace latest version with specific version
         catalog_source_data = templating.load_yaml(QE_APP_REGISTRY_SOURCE)
 
-        metallb_version = config.default_cluster_ctx.ENV_DATA.get("metallb_version")
+        # metallb_version = config.default_cluster_ctx.ENV_DATA.get("metallb_version")
```

Comment:
please uncomment this line.
this function is utility function, update `config.default_cluster_ctx.ENV_DATA.get("metallb_version")` before calling this function instead

Reply:
Addressed in latest commit.